### ISSUE TITLE
Enum wont be selected on load if it not the first element

### ISSUE THREAD

**Issue by mochouinard** [json-editor/json-editor#1558]
### General information

- json-editor version: latest 2.15.0

### Expected behavior
When you load a JSON with an enum, the correct item is selected in the drop box.

### Actual behavior
In version 2.15.0 (2.14.1 doesn't have this problem), if you load a json with enum, if it not the first element that is selected in the json, the dropdown won't have anything selected.

### Steps to reproduce the behavior
Simple schema item below and load { "example": "suzanne" }   The drop down won't have anything selected in 2.15.0.  

```javascript
{
  "title": "json schema example",
  "type": "object",
  "properties": {
    "example": {
      "type": "string",
      "description": "This is an example schema.",
        "enum": [
            "louise",
            "suzanne",
            "heather"
          ],
          "options": {
            "enum_titles": [
              "FR - Louise",
              "FR - Suzanne",
              "EN - Heather"
            ]
          }

    }
  }
}
```

**Comment by waltzofpearls** [comment]
I encountered the same issue when using `enum`, `enum_titles` and `choices` editor in 2.15.0. The json-editor always has the first item in the list selected. It works fine when I switched back to 2.14.1.

```javascript
{
  'country': {
    'title': 'Countries',
    'type': 'array',
    'format': 'table',
    'items': {
      'type': 'string',
      'title': 'Country',
      'format': 'choices',
      'enum': Object.keys(countries),
      'options': { enum_titles: Object.values(countries) }
    }
  }
}
```

**Comment by germanbisurgi** [comment]
fixed in #1565
